lib/jekyll/commands/serve.rb

```diff
@@ -2,6 +2,21 @@ module Jekyll
   module Commands
     class Serve < Command
       class << self
+        def process(opts)
```

Comment:
Rather than monkey patching another method, can we pass the options hash to our existing monkeypatch?